docs/package.json

```diff
@@ -15,8 +15,11 @@
   },
   "dependencies": {
     "codemirror": "^5.5.0",
+    "github-markdown-css": "^2.1.0",
+    "hightlight.js": "^8.9.1",
     "history": "^1.11.1",
     "intl": "^1.0.1",
+    "marked": "^0.3.5",
```

Comment:
I'll check out if the raw-loader is needed in production, otherwise, I would like to let the raw-load  staying under devDependencies.